skrf/circuit.py

```diff
@@ -182,6 +184,9 @@ def __init__(self,
             circuit's dimensions becomes less efficient compared to directly calculating it with Circuit.s_external.
             This value depends on the performance of the computer and the scale of the circuit. Default is 20.
 
+            `ignore_networks` kwarg is a tuple of Networks to be ignored during the reduction process.
+            Default is an empty tuple.
```

Comment:
I'm afraid the term 'ignored' could be confusing as one could understand that the network is simply not taken into account in the circuit while this is obviously not that. However I'm not fluent in english, so just wondering... any other advice would be appreciated.

Reply:
Perhaps `dynamic_networks` could be a more accurate and descriptive term? Given that English is not my first language, I'm open to other suggestions as well.

Reply:
Yes, why not. @FranzForstmayr ot @Vinc0110, do you have any suggestion?